docs/policies/recognizing_contributors_sponsors_and_partners.rst

```diff
@@ -0,0 +1,113 @@
+
+7. Each initiative/team/working group that is recognizing or promoting the work from current significant contributors in a release must state that it is doing so as part of each release planning meeting, and must provide the list of contributors to be recognised to the release leader at least one week prior to the GA release date.
+
+8. Each initiative/team/working group's leadership chain (leads and assistant leads) are responsible for vetting, and potentially filtering, the current significant contributor list in order to conform to the overall goal and community norms of this policy.
+
+9. Persons or companies who believe they are being unfairly precluded from being identified as a current significant contributor can contact the Project Lead to discuss and, where appropriate, resolve the issue.
+
+10. Individuals or companies so recognized may promote that fact, publish that they have been recognized, and link to the recognition. Such an individual or company may not, however, imply any endorsement of their products or services by Mautic or others based on such recognition.
+
+11. The Project Lead and Community Council reserve the right to remove any recognition at any time at their discretion.
+
+Policy on promotion of sponsors and partners
+############################################
+
+The Mautic policy on recognizing sponsors and partners on mautic.org, in the Mautic application and in releases is as follows:
+
+1. Sponsors who are contributing over $100/mth will have a logo and link listed on :xref:`Mautic.org/sponsors` for the duration of their active sponsorship.
+
+2. Sponsors who are contributing under $100/mth will have their name or organization name listed on :xref:`Mautic.org/sponsors`.
+
+3. Infrastructure sponsors who are providing a service used by the Mautic Community at free or discounted price will have a logo and link to the most relevant page on their website for as long as the membership is active, listed at :xref:`Mautic.org/sponsors`.
+   
+4. If a sponsor cancels their sponsorship or the service is no longer being used, the link will be removed as soon as notification of cancellation is received
+
+5. Event sponsors will be listed on the event websites, and will not be eligible to be listed on mautic.org unless they are a regular monthly sponsor in addition to supporting the event.
+   
+6. Partners will be announced on social media and email when they become a partner, announcing their joining of the programme.
+   
+7. Partners will be featured in the email template of the Mautic Community Newsletter with a logo and link to their partners landing page.
+   
+8. The top three partners each month will be listed on the mautic.org homepage with a link to their partners landing page.
+   
+
+Credits
+#######
+
+This policy is based on those used by several other Open Source communities including:
+
+Eclipse: https://www.eclipse.org/org/documents/Recognizing_Project_Contributions_Policy_2008.pdf
```

Comment:
These need to be link files please. you can use the name (e.g. Eclipse Foundation contributor recognition policy) as the link text. Same for the second link.